test/models/maintenance_tasks/runner_test.rb

```diff
@@ -32,6 +32,18 @@ class RunnerTest < ActiveSupport::TestCase
       end
     end
 
+    test "#run yields the Run when there is no active Run (undocumented)" do
+      run = Run.create!(task_name: @name, status: :paused)
+
+      @runner.run(name: @name) { |yielded| @run = yielded }
```

Comment:
```suggestion
      @runner.run(name: @name) { |run| @run = run }
```
Nit

Reply:
I can't really do that one because Rubocop complains.

Reply:
Right, forgot we were using `run` already above. Ignore me!